webpay/pay/views.py

```diff
@@ -127,7 +127,7 @@ def configure_transaction(request):
 
     """
 
-    form = NetCodeForm(request.POST)
+    form = NetCodeForm(data or request.POST)
```

Comment:
this fixed a pretty bad bug that the test suite did not cover. Unfortunately, adding coverage isn't so simple: https://github.com/tomchristie/django-rest-framework/issues/1928